.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# prettier
+.prettierignore
```

Comment:
Why?

Reply:
As referenced in the PR description, the Prettier VSCode extension was breaking the formatting of the `CODEOWNERS `file on save. I added a local `.prettierignore` to prevent this behavior, and didn't see any reason to include the `.prettierignore` with the PR, so added it to `.gitignore`.

Looks like it made it through anyway. And upon more thought, I could just turn prettier off locally for a minute if it happens again.

I'll remove it.

Reply:
All references to Prettier removed.